installing/installing_vsphere/upi/upi-vsphere-preparing-to-install.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upi-vsphere-preparing-to-install"]
+= Preparing to install a cluster using user-provisioned infrastructure
+include::_attributes/common-attributes.adoc[]
+:context: upi-vsphere-preparing-to-install
+
+toc::[]
+
+You prepare to install an {product-title} on vSphere by:
+
+* Downloading the installation program.
++
+[NOTE]
+====
+If you are installing in a restricted network, you extract the installation program from the mirrored content. For more information, see xref:../../../installing/disconnected_install/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[Mirroring images for a disconnected installation].
```

Comment:
I know that the install content uses "restricted network" a lot, but just an FYI that our [term glossary prefers the term "disconnected environment"](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#disconnected). Up to you if you want to be consistent with the install docs vs the term glossary:

```suggestion
If you are installing in a disconnected environment, you extract the installation program from the mirrored content. For more information, see xref:../../../installing/disconnected_install/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[Mirroring images for a disconnected installation].
```